test/spec/modules/pubwiseBidAdapter_spec.js

```diff
@@ -0,0 +1,135 @@
+// import or require modules necessary for the test, e.g.:
+
+import {expect} from 'chai';
+import {spec as adapter} from 'modules/pubwiseBidAdapter.js';
+import * as utils from 'src/utils.js';
+
+const sampleBid = {
+  bidder: 'pubwise',
+  params: {
+    siteId: 'xxxxxx',
+    spotId: '12345678',
+    isTest: true
+  },
+  sizes: [[300, 250]],
+  mediaTypes: {
+    banner: {
+      sizes: [[300, 250]]
+    }
+  },
+  bid_id: '1234',
+};
+
+const sampleRequest = {
+  bidder: 'pubwise',
+  params: {
+    siteId: 'xxxxxx',
+    puspotId: '12345678',
+    isTest: true
+  },
+  sizes: [{width: 300, height: 250}],
+  mediaTypes: {
+    banner: {
+      sizes: [[300, 250]]
+    }
+  },
+  bid_id: '1234',
+};
+
+describe('PubWiseAdapter', function () {
+  describe('Check methods existance', function () {
+    it('exists and is a function', function () {
+      expect(adapter.isBidRequestValid).to.exist.and.to.be.a('function');
+    });
+    it('exists and is a function', function () {
+      expect(adapter.buildRequests).to.exist.and.to.be.a('function');
+    });
+    it('exists and is a function', function () {
+      expect(adapter.interpretResponse).to.exist.and.to.be.a('function');
+    });
+    it('exists and is a function', function () {
+      expect(adapter.getUserSyncs).to.exist.and.to.be.a('function');
+    });
+  });
+
+  describe('Check method isBidRequestValid return', function () {
+    let bid = sampleBid;
+
+    it('should be true', function () {
+      expect(adapter.isBidRequestValid(bid)).to.be.true;
+    });
+  });
+
+  describe('Check buildRequests method', function () {
+    // Bids to be formatted
+    let bid1 = sampleBid;
+
+    let jsonData = {
+      pbdata: [sampleBid],
```

Comment:
IMO, you should not use `sampleBid` variable here, but another object which describes exactly what you expect. 
Actually your test will always pass as all is passed as reference, and if your `transformSizes()` function (for example) doesn't returns what you want, you'll not be notified.

Reply:
Let me take another look at that. We were following some existing patterns in other tests, but we basically just pass through the data from the endpoint with very little modification so the in might actually look exactly like the out when passed this way in the test, since it's mocked.